src/account.ts

```
@@ -398,6 +399,24 @@ class MasterHandle extends HDKey {
       })
     })
   }
+
+  /**
+   * creates a V3 keystore file for the master handle
+   *
+   * @param password - the password to encrypt the key with. make it strong!
+   */
+  toV3 = async (password) => {
+    if(!password) {
+      return false;
```

You should throw rather than return false